index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Concert-page</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="overall-container">
+        <div class="header">
+            <!-- header navbars starts here -->
+            <nav class="header-navbars">
+                <nav class="desktop">
+
+                    <div class="dest-nav-logo">
+                        <img src="photo/assets/logo_transparent.png" alt="">
+                    </div>
+
+                    <ul class="nav-ul">
+                        
+                        <li><a href="about.html">About</a></li>
+                        
+                        <li><a href="#tul-1">Event</a></li>
+                    
+                        <li><a href="soon.html">Join</a></li>
+                        
+                        <li><a href="soon.html">Sponsor</a></li>
+                        
+                        <li><a href="soon.html">News</a></li>
+                    </ul>
+                    <div class="cc-nav">
+                        <div class="desk-cc">CC Campaign</div>
+                      </div>
+                </nav>
```

Comment:
- [ ] Great job overall, 👍 , however your navigation is missing the top bar with black background as shown in the [behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) template containing some dummy links eg "mypage", "logout" etc.  Please add the top bar to your navigation to match the [behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) template. 👍 

Behance Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/183138344-7d5abf8d-7d2c-401a-ac9c-97478844c726.png)  |  ![image](https://user-images.githubusercontent.com/60618877/183138486-f0bea8c1-5365-4523-9a36-23ed993e294d.png)